creator-node/src/routes/tracks.js

```diff
@@ -959,6 +959,69 @@ router.get(
  * Gets a streamable mp3 link for a track by encodedId. Supports range request headers.
  * @dev - Wrapper around getCID, which retrieves track given its CID.
  **/
-router.get('/tracks/cidstream/:CID', contentAccessMiddleware, getCID)
+router.get(
```

Comment:
looks like this does not use the `shouldCache` flag which informs whether we cache or not the result

Reply:
actually, nevermind. didn't see that there is another middleware after this logic